fenix/app/src/test/java/org/mozilla/fenix/search/awesomebar/AwesomeBarViewTest.kt

```diff
@@ -852,6 +850,90 @@ class AwesomeBarViewTest {
         assertEquals(1, result.filterIsInstance<SearchEngineSuggestionProvider>().size)
     }
 
+    @Test
+    fun `GIVEN sponsored suggestions are enabled WHEN configuring providers THEN add the Firefox Suggest suggestion provider`() {
```

Comment:
I really wanted to use [`@ParameterizedTest`](https://www.petrikainulainen.net/programming/testing/junit-5-tutorial-writing-parameterized-tests/) here, but the only other example of `@ParameterizedTest` that I found was in [`mozilla-detekt-rules`](https://github.com/mozilla-mobile/firefox-android/blame/ecbf4f6fa5221d4ed0cad35a5a847ee6a9ae2243/fenix/mozilla-detekt-rules/src/test/java/org/mozilla/fenix/detektrules/perf/MozillaUseLazyMonitoredTest.kt#L60-L67), and I wasn't sure if pulling JUnit 5 into Fenix would cause other issues.